src/main/java/org/worldcoin/bukkit/plugin/worldid/tasks/CheckVerified.java

```diff
@@ -73,14 +80,18 @@ public Boolean handleResponse(final ClassicHttpResponse response) throws IOExcep
                                 throw new IllegalStateException("player is already verified");
                             }
 
-                            final RegisteredServiceProvider<LuckPerms> provider = Bukkit.getServicesManager().getRegistration(LuckPerms.class);
-                            final LuckPerms api = provider.getProvider();
+                            final LuckPerms api = LuckPermsProvider.get();
                             final User user = api.getPlayerAdapter(Player.class).getUser(player);
                             final InheritanceNode node = InheritanceNode.builder(groupName).value(true).build();
                             user.data().add(node);
                             user.setPrimaryGroup(groupName);
                             api.getUserManager().saveUser(user);
                             player.sendMessage("You've successfully verified with World ID!");
+                            posthog.capture(player.getUniqueId().toString(), "minecraft integration verification", new HashMap<String, Object>() {
```

Comment:
we need to add `getConfig().getString("server-uuid")` so we can link these verifications to the app in posthog